backend/api/Controllers/ParcelController.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Pims.Dal.Services;
+using AutoMapper;
+using Pims.Dal.Helpers.Extensions;
+using Model = Pims.Api.Models;
+using Entities = Pims.Dal.Entities;
+using Pims.Api.Models;
+
+namespace Pims.Api.Controllers
+{
+    /// <summary>
+    /// ParcelController class, provides endpoints for managing my parcels.
+    /// </summary>
+    [Authorize]
+    [ApiController]
+    [Route("/api/my/[controller]")]
+    public class ParcelController : ControllerBase
+    {
+        #region Variables
+        private readonly ILogger<ParcelController> _logger;
+        private readonly IConfiguration _configuration;
+        private readonly IParcelService _parcelService;
+        private readonly IMapper _mapper;
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a ParcelController class.
+        /// </summary>
+        /// <param name="logger"></param>
+        /// <param name="configuration"></param>
+        /// <param name="dbContext"></param>
+        public ParcelController(ILogger<ParcelController> logger, IConfiguration configuration, IParcelService parcelService, IMapper mapper)
+        {
+            _logger = logger;
+            _configuration = configuration;
+            _parcelService = parcelService;
+            _mapper = mapper;
+        }
+        #endregion
+
+        #region Endpoints
+        /// <summary>
+        /// Get all the parcels filtered by the lat/lon coords.
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        public IActionResult GetMyParcels(double neLat, double neLong, double swLat, double swLong)
+        {
+            var parcels = _parcelService.GetParcels(neLat, neLong, swLat, swLong);
```

Comment:
What I've seen in other services is a per-city view when the map is zoomed out to a certain extent. so Instead of displaying 1000pins in victoria, you display one pin with the number "1000" to indicated the number of properties. Clicking on this pin zooms you into the city. I can log a generic story about doing somethin along these lines if you like.

Reply:
We'll need a way at some point or we'll kill our db again.  Create a story and link it to an appropriate epic.